cpp/tests/quantiles/percentile_approx_test.cu

```diff
@@ -404,7 +404,8 @@ TEST_F(PercentileApproxTest, EmptyInput)
                             3,
                             cudf::test::detail::make_null_mask(nulls.begin(), nulls.end()));
 
-  CUDF_TEST_EXPECT_COLUMNS_EQUAL(*result, *expected);
+  // TODO: change percentile_approx to produce sanitary list outputs for this case.
```

Comment:
is this for a future PR?

Reply:
Yes.   Issue for it is here: https://github.com/rapidsai/cudf/issues/10856